README.md

```diff
@@ -79,6 +79,81 @@ production:
 - If you are facing errors running the 'rails db:create' ensure that the socket file(i.e mysql.sock) is present in that location.   Some possible locations where it might be present is '/run/mysqld/mysqld.sock'  or '/var/lib/mysql/mysql.sock' and mention the exact location.
 
 
+
+## Development Process 
+
+## Step 1: Leave a comment on the GitHub issue
+When you are ready to start work on a issue:
+
+1. Let us know by leaving a comment on the issue. (Also let us know later if you are no longer working on it.)
+
+## Step 2: Update local repo development branch
+Reason: Your **local repo** *master* branch needs to stay in sync with the **project repo** *master* branch.  This is because you want to have your feature branch (to be created below) to have the latest project code before you start adding code for the feature you’re working on.
+
+    git checkout master
+    git fetch upstream
+    git merge upstream/master
+
+You should not have merge conflicts in step 3, unless you’ve made changes to your local *master* branch directly (which you should not do).
+
+## Step 3: Create the feature branch in your local and github repos
+Reason: All of your development should occur in feature branches - ideally, one branch per issue solved.  This keeps your local *master* branch clean (reflecting *upstream* master branch), allows you to abandon development approaches easily if required (e.g., simply delete the feature branch), and also allows you to maintain multiple work-in-process branches if you need to (e.g., work on a feature branch while also working on fixing a critical bug - each in its own branch).
+
+1. Create a branch whose name contains the github issue number. For example, if you are going to work on issue (which is, say, a new feature for ‘forgot password’ management):
+
+        git checkout -b forgot-password
```

Comment:
The example here is missing the issue number mentioned above.

Reply:
@fpintos pls review the changes. I have added a CONTRIBUTING.md file and added a link to it from the README.md.